### PR TITLE
How to turn off features in lsp-mode/lsp-ui

### PR DESCRIPTION
**Pull request by yyoncho**
Long awaited docs

### CODE REVIEW

**Comment by yyoncho** [comment]
Direct link: https://github.com/yyoncho/lsp-mode/blob/docs-disable/docs/blog/2020/how-to-turn-off.md

**Comment by ericdallo** [comment]
Uow, what a detailed tutorial :clap:

File: mkdocs.yml [hunk 1]
```diff
@@ -105,6 +105,7 @@ nav:
   - Blog:
       - 2020:
           - '7.0 Release': blog/2020/7.0-release.md
+          - 'A guide on disabling/enabling lsp-mode features': blog/2020/how-to-turn-off.md
```
Comment (ericdallo) [inline mkdocs.yml:108]: This probably it's a tutorial now right? not a blog post

Comment (yyoncho) [reply mkdocs.yml:108]: IDK - but we have to link it from the readme, it may go in the main tree as well.

Comment (ericdallo) [reply mkdocs.yml:108]: We can link it from the FAQ section, but I didn't get it why this should not be in the tutorials section?

File: docs/blog/2020/how-to-turn-off.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+11. Modeline diagnostics statistics
+``` elisp
+(setq lsp-modeline-diagnostics-enable nil)
+```
+![Turn Off 3](images/turn-off-3.png "turn off 3")
+
+12. Signature help
+``` elisp
+(setq lsp-signature-auto-activate nil) ;; you could manually requiest them via `lsp-signature-activate`
```
Comment (kiennq) [inline docs/blog/2020/how-to-turn-off.md:96]:
```suggestion
(setq lsp-signature-auto-activate nil) ;; you can manually request them via `lsp-signature-activate`
```

**Review by kiennq — APPROVED** [review]

File: docs/tutorials/how-to-turn-off.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# A guide on disabling/enabling lsp-mode features
+
+Early in the project, we decided to auto-configure/enable all of the stable
+features by default in order to improve discoverability and simplify the setup
+for the beginners. This decision was considered good by the majority of the
+users coming from IDEs but still in `Emacs` community there are a lot of people
+looking for a minimal distraction-free experience especially traditional long
+time users. Although the turn off `defcustom` settings are documented in
+corresponding `README` and available via `customize-group` it turned out that it
+is not very easy to find the them when you want to turn a particular feature off
```
Comment (zoechi) [inline docs/tutorials/how-to-turn-off.md:14]: redundant "the"

**Pull request merged**